### ISSUE TITLE
Increase token length

### ISSUE THREAD

**Issue by iNDicat0r** [bshaffer/oauth2-server-php#765]
I would like to increase the token length from 40 to a bigger value.  I think 40 is a very small value considering the case sensitiveness and today's huge amount of data.

Storage shouldn't be a problem so i would like to be safe than sorry and generate big access token which reduce attacks such as brute force etc.. I am also aware of mitigating these attacks by limit rate etc but i still want a solid architecture.

Regards

**Comment by bshaffer** [comment]
It would be great to make this a config value that can be modified. Submissions are welcome!

**Comment by iNDicat0r** [comment]
I will take a look. DB structure should be also changed, access_token is of type varchar(400)

**Comment by kosciuk** [comment]
And if I use JWT the token length is 600 ...

**Comment by svycka** [comment]
@kosciuk why would you want to store JWT token?

**Comment by kosciuk** [comment]
@svycka Ups! You're right, I think I must use Memory storage instead Pdo. But in http://bshaffer.github.io/oauth2-server-php-docs/overview/jwt-access-tokens/, section Using Secondary Storage, Pdo will truncate the access token

**Comment by svycka** [comment]
that's true but you could always change table to allow longer access_tokens if you really need to save (but don't know why would you) this lib does not force you to use default schema.
